js/private/yq.bzl

```diff
@@ -0,0 +1,24 @@
+"Utilities for using the yq toolchain in a repository rule"
```

Comment:
@kormide does this file belong in bazel-lib instead?

Reply:
Yeah this should go in bazel-lib.